src/responder/pam/pamsrv_passkey.c

```diff
@@ -489,6 +489,20 @@ errno_t process_passkey_data(TALLOC_CTX *mem_ctx,
         goto done;
     }
 
+    /* Don't try to process smartcard of sshpubkey mapping data */
```

Comment:
I'd write something more generic, like
```
The attribute may contain other mapping data not related to passkey. In that case, let's omit it.
```

Reply:
I don't mind the wording but I also want to be explicit about what data may be stored in SYSDB_USER_PASSKEY. I changed it to the following, let me know if you still want it modified.

~~~
    /* This attribute may contain other mapping data unrelated to passkey. In that case
     * let's omit it. For example AD user altSecurityIdentities attribute may store ssh
     * public key or smart card mapping data */
~~~

Reply:
Looks good!